src/FirstSteps/RPG/Game.cs

```diff
@@ -32,38 +24,21 @@ public static void CreateHero()
             _hero = HeroesCreator.Create(name, race);
 
         }
-        public static void HandleCommand(string command)
+        public static void HandleAdventures(string command)
         {
-            switch (command)
+           var adventure = _adventuresRegistry.GetAdventure(command);
+            if(adventure != null)
             {
-                case "stats":
-                    DisplayHeroStats();
-                    break;
-                case "treasure adventure":
-                    _treasureChest.Enter(_hero);
-                    break;
-                case "inventory":
-                    _inventory.DisplayAllItems(_hero);
-                    break;
-                case "forest":
-                    _forest.Enter(_hero);
-                    break;
-                case "dungeons":
-                    _dungeons.Enter(_hero);
-                    break;
-                case "mine":
-                    _mine.Enter(_hero);
-                    break;
-                case "village":
-                    _village.Enter(_hero);
-                    break;
-                case "hell":
-                    _hell.Enter(_hero);
-                    break;
-                default:
-                    Display.WarningText($"Command {command} not recognized");
-                    break;
+                adventure.Enter(_hero);
             }
+            else if(command == "Stats")
+            {
+                Game.DisplayHeroStats();
+            }
+            else
+            {
+                Game.GetMenu();
```

Comment:
You should not need `Game.GetMenu();` because everything is inside the `do while` loop. It would be best to display info about the unrecognized command. Like `Display.WarningText($"Command {command} not recognized");`